auth/api/iam/api.go

```diff
@@ -864,28 +880,27 @@ func (r Wrapper) CreateAuthorizationRequest(ctx context.Context, client did.DID,
 		if err != nil {
 			return nil, fmt.Errorf("failed to resolve key for signing authorization request: %w", err)
 		}
-		// default claims for JAR
-		params[jwt.IssuerKey] = client.String()
-		params[jwt.AudienceKey] = server.String()
-		params[oauth.ClientIDParam] = client.String()
-		// added by default, can be overriden by the caller
-		params[oauth.NonceParam] = cryptoNuts.GenerateNonce()
-
-		// additional claims can be added by the caller
-		modifier(params)
-
-		paramsAny := make(map[string]any, len(params))
-		for k, v := range params {
-			paramsAny[k] = v
-		}
-
-		token, err := r.jwtSigner.SignJWT(ctx, paramsAny, nil, keyId.String())
+		requestObjectParams := createRequestObject(client, server, modifier)
+		// TODO: signature type produced here must be in metadata.RequestObjectSigningAlgValuesSupported
+		token, err := r.jwtSigner.SignJWT(ctx, requestObjectParams, nil, keyId.String())
 		if err != nil {
 			return nil, fmt.Errorf("failed to sign authorization request: %w", err)
 		}
+		uriID := cryptoNuts.GenerateNonce()
```

Comment:
`uriID` -> `requestID` or `requestObjectID`?

Reply:
now `requestID` to match the API